src/utils/wallet.js

```diff
@@ -208,6 +208,7 @@ class Wallet {
             const state = await this.getAccount(this.accountId).state()
             this.twoFactor = new TwoFactor(this)
             const has2fa = this.has2fa = await this.twoFactor.isEnabled()
+            console.log('loadAccount has2fa', this.has2fa);
```

Comment:
Do you want to keep the logs?

Reply:
yes, in case I need to debug further in prod